scripts/lab/sweep.sh

```
@@ -0,0 +1,72 @@
+  run() {
+    if [ "$RUNNER" = vitest ]; then
+      # --dir tests: a bare path filter also matches copies under .claude/worktrees/ (one game per copy)
+      env $benv PARAMS="$params" MIN="$MINUTES" SPAWN="$sp" LAB_OUT="$OUT" OUTFILE="p_${name}_${batch}_${sp}.txt" TAG="${name}_${batch}" \
+        npx vitest --dir tests tests/lab/playbook.lab.test.ts --run > "$OUT/.err_${name}_${batch}_${sp}" 2>&1
+    else
+      env $benv PARAMS="$params" MIN="$MINUTES" SPAWN="$sp" LAB_OUT="$OUT" OUTFILE="p_${name}_${batch}_${sp}.txt" TAG="${name}_${batch}" \
+        node --import tsx tests/lab/playbook.lab.ts > "$OUT/.err_${name}_${batch}_${sp}" 2>&1
+    fi
+  }
+  if run || { echo "retry $name $batch $sp"; run; }; then rm -f "$OUT/.err_${name}_${batch}_${sp}"; echo "done $name $batch $sp"
+  else echo "FAILED $name $batch $sp (see $OUT/.err_${name}_${batch}_${sp})"; fi' _ {}
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)